src/main/java/com/google/firebase/messaging/ApnsFcmOptions.java

```diff
@@ -0,0 +1,67 @@
+package com.google.firebase.messaging;
```

Comment:
Add license header and make the class final.

Reply:
done